test/over_react/component_declaration/component_type_checking_test.dart

```diff
@@ -96,24 +98,34 @@ main() {
       TwoLevelWrapper: TwoLevelWrapper2,
     );
   });
-}
 
-testComponentTypeChecking({
```

Comment:
This shared suite still exists, I just moved some tests out of it that didn't make sense to be shared